en-US/firefox/privacy-hub.lang

```diff
@@ -0,0 +1,223 @@
+## NOTE: Demo page at https://www-demo1.allizom.org/firefox/privacy/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/privacy/
+
+
+;Privacy
+Privacy
+
+
+;Our Promise
+Our Promise
+
+
+;Our Products
+Our Products
+
+
+;Read the <a href="%s">Privacy Notice</a> for our products
+Read the <a href="%s">Privacy Notice</a> for our products
+
+
+# These strings are used in /firefox/privacy/
+# HTML page title
+;Firefox Privacy Promise
+Firefox Privacy Promise
+
+
+# HTML page description
+;Firefox takes less data, keeps it safe, and with no secrets.
+Firefox takes less data, keeps it safe, and with no secrets.
+
+
+;Firefox products are designed to protect your <strong>privacy</strong>
```

Comment:
Is the <strong> used for the entire sentence or just one word? The demo is for the whole sentence.

Reply:
The `<strong>` tag is a placeholder for a CSS styling hook.